core/templates/services/rte-helper-modal.controller.spec.ts

```diff
@@ -158,4 +159,71 @@ describe('RteHelperModalComponent', () => {
       flush();
     }));
   });
+  describe('when there are validation errors in link form control', function() {
+    var customizationArgSpecs = [{
+      url: 'oppia.org',
+      text: 'oppia',
+    }];
+
+    beforeEach(() => {
+      fixture = TestBed.createComponent(RteHelperModalComponent);
+      component = fixture.componentInstance;
+      component.componentId = 'link',
+      component.attrsCustomizationArgsDict = {
+        heading: 'This value is not default.'
```

Comment:
I don't think links have headings. Why does this test still work?

Reply:
done